group-coordinator/src/test/java/org/apache/kafka/coordinator/group/OffsetMetadataManagerTest.java

```diff
@@ -183,20 +183,21 @@ public List<OffsetFetchResponseData.OffsetFetchResponseTopics> fetchOffsets(
             long committedOffset
         ) {
             return offsetMetadataManager.fetchOffsets(
-                groupId,
-                topics,
+                new OffsetFetchRequestData.OffsetFetchRequestGroup()
+                    .setGroupId(groupId)
+                    .setTopics(topics),
                 committedOffset
-            );
+            ).topics();
```

Comment:
nit: is there a reason we left this one as a list of topics and not the resonse?

Reply:
It found it easier to use in tests like this. Note that I updated the code to verify that the group id in the response is set correctly.